access/slackbot/install

```diff
@@ -5,10 +5,18 @@
 #
 BINDIR=/usr/local/bin
 
+#
+# the directory where Teleport Plugins store their data files
+# and certificates
+#
+DATADIR=/var/lib/teleport/plugins/slackbot
+
 [ ! $(id -u) != "0" ] || { echo "ERROR: You must be root"; exit 1; }
 cd $(dirname $0)
 mkdir -p $BINDIR
 cp -f slackbot $BINDIR/ || exit 1
 
+mkdir -p $DATADIR
```

Comment:
You forgot `|| exit 1` part :)

Reply:
Also it's better to place this command *after* the `Teleport Slackbot binaries have been copied to $BINDIR` message.

Reply:
Either you could do `mkdir $BINDIR $DATADIR` at once. It case of failure, `mkdir` prints what particular write operation haven't succeed

Reply:
Thanks for catching this, it'd make the script more elegant.